velox/expression/ExprToSubfieldFilter.h

```diff
@@ -15,42 +15,45 @@
  */
 #pragma once
 
-#include "velox/type/Filter.h"
+#include <velox/core/Expressions.h>
+#include <velox/core/ITypedExpr.h>
+#include <velox/type/Filter.h>
+#include <velox/type/Subfield.h>
 
-namespace facebook::velox::common::test {
+namespace facebook::velox::exec {
 
-inline std::unique_ptr<common::BigintRange> lessThan(
+inline std::unique_ptr<velox::common::BigintRange> lessThan(
```

Comment:
Why do we need "velox::" prefix here?

Reply:
Multiple of filter builder functions take Expr as input for example:
```
std::unique_ptr<common::Filter> makeLessThanFilter(
    const core::TypedExprPtr& upperExpr)
```

Reply:
> Multiple of filter builder functions take Expr as input for example:

They take core::xxxExpr, which are defined in "core", not in "expression".